src/Components/about/about.css

```diff
@@ -1,29 +1,24 @@
-.about{
-    background-color: lightblue;
-    display: flex;
-     
+.about {
+  background-color: lightblue;
+  display: flex;
 }
 
-.about-left{
+.about-left {
   flex: 1;
-  background-image: url('C:\Users\AndrewP\Dropbox\Code\Portfolio Website\portfolioapp\src\img\0606201317b.png');
+  background-image: url('../../img/0606201317b.png');
```

Comment:
I'm guessing if this doesn't work for you, than there is a file traversal difference when rendering css (but not js?). 

I dunno, try it.